extensions/blocks/podcast-player/components/audio-player.js

```diff
@@ -0,0 +1,72 @@
+/* global _wpmejsSettings, MediaElementPlayer */
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+
+const meJsSettings = typeof _wpmejsSettings !== 'undefined' ? _wpmejsSettings : {};
+
+class AudioPlayer extends Component {
+	audioRef = el => {
+		if ( el ) {
+			// Construct audio element.
+			this.audio = document.createElement( 'audio' );
+			this.audio.src = this.props.initialTrackSource;
```

Comment:
Could we use `setAudioSource` here?

Reply:
I was thinking about it but it looked a bit weird to break the `this.audio.` flow for this. I can update it if you'd like to.